inflate.c

```diff
@@ -1266,7 +1232,7 @@ int32_t Z_EXPORT PREFIX(inflateCopy)(PREFIX3(stream) *dest, PREFIX3(stream) *sou
     copy->next = copy->codes + (state->next - state->codes);
     if (window != NULL) {
         wsize = 1U << state->wbits;
-        memcpy(window, state->window, wsize);
+        memcpy(window, state->window, (wsize + state->chunksize) * 2);
```

Comment:
Like above... Only `wsize` needs to be multiplied by `2`.

Reply:
Fixed.